tsup.config.ts

```diff
@@ -0,0 +1,59 @@
+import { defineConfig } from "tsup";
```

Comment:
Pretty sexy, the configuration style here is nice. Looks good to me assuming everyone is happy with the bundling and source map stuff.